main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -761,6 +766,17 @@ private void notifyDataSetChanged() {
         progress.dismiss();
 
         Settings.addCacheToHistory(cache.getGeocode());
+        final ScrollView scrollNew = ButterKnife.findById(this, R.id.detailScroll);
+        if ((scroll != null) && (scrollNew != null)) {
```

Comment:
someone removed / renamed the detailScroll ? 
inner parenthesis are for readability. 
I can remove the statement if you think it's not needed...